common.js

```diff
@@ -1708,6 +1708,17 @@ const allSoloCars = {
       'all': ['ssp'],
     },
   },
+  'Sunbeam': {
+    'Tiger': {
+      'all': ['es'],
+    },
+    'Tiger (260, 289)': {
```

Comment:
This is the same as the regular Tiger and should be merged with it. They only made two versions of the Tiger. See https://en.wikipedia.org/wiki/Sunbeam_Tiger.

Reply:
got it